bundles/org.openhab.binding.bmwconnecteddrive/src/main/java/org/openhab/binding/bmwconnecteddrive/internal/handler/RemoteServiceHandler.java

```diff
@@ -99,24 +108,39 @@ public RemoteServiceHandler(VehicleHandler vehicleHandler, ConnectedDriveProxy c
         handler = vehicleHandler;
         proxy = connectedDriveProxy;
         final VehicleConfiguration config = handler.getConfiguration().get();
-        serviceExecutionAPI = proxy.baseUrl + config.vin + proxy.serviceExecutionAPI;
-        serviceExecutionStateAPI = proxy.baseUrl + config.vin + proxy.serviceExecutionStateAPI;
+        legacyServiceExecutionAPI = proxy.baseUrl + config.vin + proxy.serviceExecutionAPI;
+        legacyServiceExecutionStateAPI = proxy.baseUrl + config.vin + proxy.serviceExecutionStateAPI;
+        serviceExecutionAPI = proxy.remoteCommandUrl + config.vin + "/";
+        serviceExecutionStateAPI = proxy.remoteStatusUrl;
     }
 
     boolean execute(RemoteService service, String... data) {
         synchronized (this) {
             if (serviceExecuting.isPresent()) {
+                logger.debug("Execution rejected - {} still pending", serviceExecuting.get());
                 // only one service executing
                 return false;
             }
             serviceExecuting = Optional.of(service.name());
         }
-        final MultiMap<String> dataMap = new MultiMap<String>();
-        dataMap.add(SERVICE_TYPE, service.name());
-        if (data.length > 0) {
-            dataMap.add(DATA, data[0]);
+        if (myBmwApiUsage) {
+            final MultiMap<String> dataMap = new MultiMap<String>();
+            if (data.length > 0) {
+                dataMap.add(DATA, data[0]);
+                proxy.post(serviceExecutionAPI + service.name().toLowerCase().replace("_", "-"),
```

Comment:
Assuming `RemoteService` is an enum it might make sense to add this repeated conversion there:

```java
service.name().toLowerCase().replace("_", "-")

Reply:
Right - done